frontend/src/modules/processes/actions.js

```diff
@@ -0,0 +1,57 @@
+import { listPropertyValues } from "../experimentRuns/actions";
+import {createNetworkActionTypes, networkAction} from "../../utils/actions";
+import api from "../../utils/api";
+
+export const GET = createNetworkActionTypes("PROCESSES.GET");
+export const LIST = createNetworkActionTypes("PROCESSES.LIST");
+
+export const get = id => async (dispatch, getState) => {
+    const process = getState().processes.itemsByID[id];
+    if (process && process.isFetching)
+        return;
+
+    return await dispatch(networkAction(GET, api.processes.get(id), { meta: { id } }));
+};
+
+export const list = (options) => async (dispatch, getState) => {
+    if (getState().processes.isFetching)
+        return;
+
+    return await dispatch(networkAction(LIST,
+        api.processes.list(options),
+        { meta: { ...options} }
+    ));
+};
+
+export const listProperties = (id) => async (dispatch, getState) => {
+    if (getState().propertyValues.isFetching)
+        return;
+
+    const { itemsByID: processesByID } = getState().processes    
+    const { itemsByID: propertyValuesByID } = getState().propertyValues
```

Comment:
aren't you creating twice the same constant with 2 different values ?
I don't know this notation but to my understanding what you do is const itemsByID = ... and then const itemsByID = ...

Reply:
that actually would be much simpler 😂